ModularTegustation/tegu_items/representative/research/kcorp.dm

```diff
@@ -0,0 +1,85 @@
+
+//-----K_CORP-----
+//K Corp sells mostly healing gear.
+///Healing Items Line
+/datum/data/lc13research/syringe
+	research_name = "Ampule Runoff Permit"
+	research_desc = "Due to your efforts, we are granting you the privilage of <br>purchasing low level hp ampules at a 90% discount."
+	cost = AVERAGE_RESEARCH_PRICE
+	corp = K_CORP_REP
+
+/datum/data/lc13research/syringe/ResearchEffect(obj/structure/representative_console/caller)
+	ItemUnlock(caller.order_list, "K Corp Ampule ",	/obj/item/ksyringe, 300)
+	..()
+
+/datum/data/lc13research/medbeam
+	research_name = "K Corp Experimental Beamgun "
+	research_desc = "L Corp Branch Representative. <br>I have a new breakthrough in our technology regarding the nanomachines. <br>This item will fire a stream of continuous Kcorp syringe fluid at a distance.  <br>Use it wisely.  <br>- Jill, from research."
+	cost = AVERAGE_RESEARCH_PRICE+10
+	corp = K_CORP_REP
+	required_research = /datum/data/lc13research/syringe
+
+/datum/data/lc13research/medbeam/ResearchEffect(obj/structure/representative_console/caller)
+	new /obj/item/gun/medbeam(get_turf(caller))
```

Comment:
No changes to this means it can heal people who are working and a significant amount at that.

Reply:
RIGHT. I forgot about that.